File: source/code/nodejs/account_management/account_linking_policy.js
// You can enable an Application's Account Linking Policy

var application; // An application, fetched from Client.getApplication();

application.getAccountLinkingPolicy(function(err, AccountLinkingPolicy) {
  if (err) {
    return console.error(err);
  }

  AccountLinkingPolicy.status = 'ENABLED';

  AccountLinkingPolicy.save(function (err, updatedResource) {
    if (err) {
      return console.error(err);
    }

    console.log('Enabled Account Linking Policy: ', updatedResource);
  });
});

// You can also enable an Organization's Account Linking Policy

var organization; // A previously fetched Organization object

organization.getAccountLinkingPolicy(function(err, AccountLinkingPolicy) {
  if (err) {
    return console.error(err);
  }

  AccountLinkingPolicy.status = 'ENABLED';

  AccountLinkingPolicy.save(function (err, updatedResource) {
    if (err) {
      return console.error(err);
    }

    console.log('Enabled Account Linking Policy: ', updatedResource);
  });
});

File: source/code/ruby/account_management/account_linking_policy.rb
application = client.applications.get(application_href)

account_linking_policy = application.account_linking_policy

account_linking_policy.status = 'ENABLED'
account_linking_policy.save

# You can also enable an Organization's Account Linking Policy

organization = client.organizations.get(organization_href)

account_linking_policy = organization.account_linking_policy

account_linking_policy.status = 'ENABLED'
account_linking_policy.save
